scss/_settings_colors.scss

```diff
@@ -108,10 +105,8 @@ $colors--dark-theme--border-default: rgba($colors--dark-theme--text-default, 0.2
 $colors--dark-theme--border-high-contrast: rgba($colors--dark-theme--text-default, 0.4) !default;
 $colors--dark-theme--border-low-contrast: rgba($colors--dark-theme--text-default, 0.1) !default;
 
-$colors--dark-theme--nav-highlight: $color-navigation-highlight !default;
-
 // Branding colors
 $color-brand: #333 !default;
 $color-brand-background: $colors--dark-theme--background-default;
-$color-accent: $color-brand !default;
+$color-accent: #e95420 !default;
 $color-accent-background: $color-accent !default;
```

Comment:
We probably still want this one as `$color-brand` as that was the previous experience.

Reply:
do you mean the $color-accent-background?